### PR TITLE
feat: Commerce Quotes Scaffold

### PR DESCRIPTION
**Pull request by pawelfras**

### CODE REVIEW

File: storage/.verdaccio-db.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"list":["@spartacus/styles","@spartacus/schematics","@spartacus/asm","@spartacus/assets","@spartacus/cart","@spartacus/cdc","@spartacus/cds","@spartacus/checkout","@spartacus/commerce-quotes","@spartacus/core","@spartacus/digital-payments","@spartacus/epd-visualization","@spartacus/order","@spartacus/organization","@spartacus/product-configurator","@spartacus/product","@spartacus/qualtrics","@spartacus/setup","@spartacus/smartedit","@spartacus/storefinder","@spartacus/storefront","@spartacus/tracking","@spartacus/user"],"secret":"9897281102d13faf48c0d66daf87fed19a20506857f684b280d8d3f5a5b87768"}
```
Comment (psmul) [inline storage/.verdaccio-db.json]: We don't want to commit verdaccios files

File: feature-libs/commerce-quotes/.release-it.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "plugins": {
+    "../../scripts/release-it/bumper.js": {
+      "out": [
+        {
+          "file": "package.json",
+          "path": [
+            "peerDependencies.@spartacus/core",
+            "peerDependencies.@spartacus/storefront",
+            "peerDependencies.@spartacus/schematics",
+            "peerDependencies.@spartacus/styles"
```
Comment (psmul) [inline feature-libs/commerce-quotes/.release-it.json]: I think for now I would leave only schematics in peers. We're not certain if other libs will be required

File: feature-libs/commerce-quotes/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "@angular/common": "^13.2.0",
+    "@angular/core": "^13.2.0",
+    "@angular/forms": "^13.2.0",
+    "@angular/router": "^13.2.0",
+    "@ng-bootstrap/ng-bootstrap": "^11.0.0",
+    "@ng-select/ng-select": "^8.0.0",
+    "@spartacus/core": "4.1.0-next.0",
+    "@spartacus/schematics": "4.1.0-next.0",
+    "@spartacus/storefront": "4.1.0-next.0",
+    "@spartacus/styles": "4.1.0-next.0",
```
Comment (psmul) [inline feature-libs/commerce-quotes/package.json:37]: For now I would only leave `@spartacus/schematics` as we're not sure if we will need other libs

File: package.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
...
-    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && yarn build:tracking  && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cdc yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup",
+    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && yarn build:tracking  && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cdc yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup && yarn build:commerce-quotes",
```
Comment (psmul) [inline package.json]: After some thinking, I would put it before we build setup lib (or maybe even remove `build:setup` as it's obsolete library) and put it right after visualization lib

**Review by psmul — CHANGES_REQUESTED** [review]
Beside of commets, please run commands below, which are required for our validation checks on pipeline:
1. `yarn prettier:fix`
2. `yarn config:update`
3. `yarn generate:deps`

File: feature-libs/commerce-quotes/.release-it.json (hunk 2 above)
Comment (pawelfras) [reply feature-libs/commerce-quotes/.release-it.json]: I've checked which peers are used in `commerce-quotes` feature-lib and only `@spartacus/storefront` is not used. I'll remove it.

File: feature-libs/commerce-quotes/package.json (hunk 3 above)
Comment (pawelfras) [reply feature-libs/commerce-quotes/package.json:37]: I've checked which peers are used in commerce-quotes feature-lib and only @spartacus/storefront is not used. I'll remove it.

**Review by szczepaniak-michal — APPROVED** [review]

**Pull request merged**